dedupe/record_linkage_testing/scripts/test.sh

```diff
@@ -66,3 +66,11 @@ EOF
 )
 
 curl -X POST http://api:8080/link-record -d "${EXAMPLE_FHIR}" --header "Content-Type: application/json"
+
+# Run Synthea with parameters for seeding, the population size and the output directory.
+# Additionally disable generating hospital and practitioner FHIR resources, as we only
+# want patient data and limit the results to only alive patients.
+# For a full list of parameters, see the [configuration file](https://github.com/synthetichealth/synthea/blob/master/src/main/resources/synthea.properties).
+java -jar synthea.jar --exporter.hospital.fhir.export=false --exporter.practitioner.fhir.export=false \
+    --generate.only_alive_patients=true \
+    -p "${POPULATION_SIZE}" -s "${SEED}" -cs "${CLINICIAN_SEED}" -o "${SYNTHEA_OUTPUT_DIR}"
```

Comment:
Do we have examples of the output records?

Reply:
Synthea has a sample repo with some zip files that you can download to take a look.  Try this one https://github.com/synthetichealth/synthea-sample-data/blob/main/downloads/latest/synthea_sample_data_fhir_latest.zip